jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -526,6 +533,37 @@ boolean jarRepackagedBySpringBoot() {
   @Override
   public JibContainerBuilder runPluginExtensions(JibContainerBuilder jibContainerBuilder)
       throws JibPluginExtensionException {
-    return jibContainerBuilder;
+    return runPluginExtensions(
+        ServiceLoader.load(JibMavenPluginExtension.class).iterator(), jibContainerBuilder);
```

Comment:
I wonder if we need to create some sort of ordering mechanism so like some file manipulation extension can run *after* a framework extension has populated all the file?

Reply:
Good idea. In a follow-up PR, I'll add a new extension configuration option, as I proposed in the internal doc. I think I will be able to enforce ordering with it.